package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ThreatConnect",
-  "version": "3.6.3",
+  "version": "3.6.4",
```

Comment:
I'd say this counts as a new feature rather than just a bug fix so I would bump the minor version here (`3.7.0`)